src/core/services/VaultService.ts

```diff
@@ -58,9 +58,10 @@ export class VaultServiceImpl implements VaultService {
   /* -------------------------------------------------------------------------- */
 
   public async getSupportedVaults({ network, addresses }: GetSupportedVaultsProps): Promise<Vault[]> {
+    const { YVBOOST } = this.config.CONTRACT_ADDRESSES;
```

Comment:
Would be nice to have a // TODO

Reply:
done https://github.com/yearn/yearn-finance-v3/pull/577/commits/e63a99fccd8069eb1ab24bd6e1e4736591550419